gyp/platform-ios.gypi

```diff
@@ -61,6 +61,7 @@
         '../platform/ios/NSString+MGLAdditions.m',
         '../platform/ios/vendor/SMCalloutView/SMCalloutView.h',
         '../platform/ios/vendor/SMCalloutView/SMCalloutView.m',
+        '../platform/ios/resources/',
```

Comment:
Although, I didn't notice this before — where does it come from? It doesn't include `README` so is it the unbundled `.bundle`?

<img width="267" alt="screen shot 2015-09-02 at 1 53 04 pm" src="https://cloud.githubusercontent.com/assets/1198851/9639423/f7badf34-5179-11e5-899f-993582fa86e8.png">